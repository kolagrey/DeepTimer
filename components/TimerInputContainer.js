import React from 'react';
import { View } from 'react-native';
import styles from '../styles';

const TimerInputContainer = (props) => {
    return (
        <View style={styles.textArea}>
            {props.children}
        </View>
    );
};

export default TimerInputContainer;