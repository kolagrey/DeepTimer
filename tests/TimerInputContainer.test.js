import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';

import TimerInputContainer from '../components/TimerInputContainer';

describe('<TimerInputContainer/>', () => {

    test('renders correctly', () => {
        const tree = renderer.create(<TimerInputContainer/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it("is of type 'View'", () => {
        const tree = renderer
            .create(
                <TimerInputContainer />,
            ).toJSON();
        expect(tree.type).toBe('View');
    });

    it('can hold a child', () => {
        const tree = renderer
            .create(
                <TimerInputContainer>
                    <Text>
                        Hello Test
                    </Text>
                </TimerInputContainer>,
            ).toJSON();
        expect(tree.children.length).toBe(1)
    });
});