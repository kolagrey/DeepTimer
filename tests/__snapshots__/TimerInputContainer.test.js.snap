// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimerInputContainer/> renders correctly 1`] = `
<View
  style={
    Object {
      "width": 200,
    }
  }
/>
`;
